react-frontend/src/components/Guides/guides.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Query from '../Query';
+import { Col, Row, Grid } from 'react-flexbox-grid';
+
+import InfoCard from '../Home/infocard';
+import SimpleBanner from '../SimpleBanner/simpleBanner';
+import GUIDES_QUERY from '../../queries/guide/guides';
+
+const GuidesPage = () => {
+  return (
+    <div className="pageContainer">
+      <SimpleBanner
+        title="Useful Guides"
+        description="It's dangerous to go it alone, take these."
```

Comment:
couldn't understand the description, maybe missing a word or two here.

Reply:
That's embarrassing on my part, that was filler text that I forgot to replace with real text. Good catch.